src/network/meson.build

```diff
@@ -51,6 +51,8 @@ sources = files('''
         netdev/macsec.h
```

Comment:
so, does this commit just move things? or also change the code? if the latter, any chance this could be spit in two, to make it easier to review? i.e. if this just moves things around I trust everything is in order, i just want to review the actual code changes afte rthat